vendors/nordic/boards/nrf52840-dk/aws_tests/config_files/iot_test_common_io_config.h

```diff
@@ -33,28 +33,31 @@
 #include "iot_spi.h"
 #include "nrf_drv_spi.h"
 
+/*------------------------FLASH-------------------------------*/
+#define IOT_TEST_COMMON_IO_FLASH_SUPPORTED    0
```

Comment:
I think these test generation flags is a bad idea and makes the test suite much more complex that required.